src/dropdown.ts

```diff
@@ -58,6 +60,7 @@ declare global {
  * @slot {Element | string} [description] - Additional information or context
  * @slot {Element} [icon:value] - Icons for the selected Dropdown Option(s). Slot one icon per Dropdown Option. `<value>` should be equal to the `value` of each Dropdown Option.
  *
+ * @fires {CustomEvent} add
```

Comment:
I made a note to try to modify our manifest scripts so the event's type is included here.